plugins/Toolbox/src/Toolbox.py

```diff
@@ -169,6 +169,27 @@ def _onAppInitialized(self) -> None:
             "materials_showcase": QUrl("{base_url}/showcase".format(base_url=self._api_url))
         }
 
+    OLD_PLUGINS = ['3DPrinterOS',
+                   'BarbarianPlugin',
+                   'CuraSolidWorksPlugin',
+                   'MakePrintablePlugin',
+                   'OctoPrintPlugin',
+                   'OrientationPlugin',
+                   'ZOffsetPlugin',
+                   'cura-siemensnx-plugin']
```

Comment:
Forcing the user to upgrade is not a terrible solution but still not as clean as just using the sdk_version tag as it's meant to be used. Now the user will still have to upgrade the existing plugin despite them having the same code.

So unfortunately in my mind this doesn't do much other than make the code messy.

We COULD scan the plugins dir though and check if any plugin folders (which are IDs) match something on the server and then offer an upgrade. That would be a bit cleaner than hard-coding these 8 IDs into the toolbox.